scripts/docker_files/docker_file_ci_ubuntu_20_04/DockerFile

```diff
@@ -47,12 +47,21 @@ RUN apt-get update -y && apt-get upgrade -y && \
     pip3 install \
         numpy \
         scipy && \
+    # install MMG 5.4
     git clone -b 'v5.4.1' --depth 1 https://github.com/MmgTools/mmg /tmp/mmg && \
     mkdir /tmp/mmg/build && \
     cd /tmp/mmg/build && \
     cmake .. -DLIBMMG3D_SHARED=ON -DLIBMMG2D_SHARED=ON -DLIBMMGS_SHARED=ON -DLIBMMG_SHARED=ON && \
     make install && \
     rm -r /tmp/mmg && \
+    # install MMG 5.5
+    git clone -b 'v5.5.1' --depth 1 https://github.com/MmgTools/mmg /tmp/mmg_5_5_1 && \
+    mkdir /tmp/mmg_5_5_1/build && \
+    cd /tmp/mmg_5_5_1/build && \
+    cmake .. -DLIBMMG3D_SHARED=ON -DLIBMMG2D_SHARED=ON -DLIBMMGS_SHARED=ON -DLIBMMG_SHARED=ON && \
+    make install && \
```

Comment:
@loumalouomega I meant if this is ok for 5.5 too?

Reply:
Yes